test/integration/util/minikube_runner.go

```diff
@@ -0,0 +1,325 @@
+/*
+Copyright 2016 The Kubernetes Authors All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package util
+
+import (
+	"bufio"
+	"bytes"
+	"context"
+	"fmt"
+	"os/exec"
+	"path/filepath"
+	"regexp"
+	"strings"
+	"sync"
+	"testing"
+	"time"
+
+	"github.com/docker/machine/libmachine/state"
+	"github.com/pkg/errors"
+	"k8s.io/minikube/pkg/minikube/assets"
+	commonutil "k8s.io/minikube/pkg/util"
+)
+
+// MinikubeRunner runs a command
+type MinikubeRunner struct {
+	Profile      string
+	T            *testing.T
+	BinaryPath   string
+	GlobalArgs   string
+	StartArgs    string
+	MountArgs    string
+	Runtime      string
+	TimeOutStart time.Duration // time to wait for minikube start before killing it
+}
+
+// Copy copies a file
+func (m *MinikubeRunner) Copy(f assets.CopyableFile) error {
+	path, _ := filepath.Abs(m.BinaryPath)
+	cmd := exec.Command("/bin/bash", "-c", path, "ssh", "--", fmt.Sprintf("cat >> %s", filepath.Join(f.GetTargetDir(), f.GetTargetName())))
```

Comment:
Is this function used? If not, remove it.

Reply:
it is not used. but would have helped one of our current PRs if it was working ...